html/js/twitter.js

```diff
@@ -324,7 +332,11 @@ QB.Phone.Notifications.LoadHashtagMessages = function(Tweets) {
             var TwitterHandle = Tweet.firstName + ' ' + Tweet.lastName
             var PictureUrl = "./img/default.png"
             if (Tweet.picture !== "default") {
-                PictureUrl = Tweet.picture
+                var regex = /^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_\+.~#?&\/=]*)\.(jpg|png)$/;
```

Comment:
Let's avoid duplicating this regex pattern

Reply:
I have updated the branch and edited the twitter.js file.
I made a function that checks the profile picture.